packages/compass-aggregations/src/components/pipeline-toolbar/pipeline-header/pipeline-actions.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import type { ConnectedProps } from 'react-redux';
+import {
+  Button,
+  css,
+  cx,
+  spacing,
+  Link,
+  Icon,
+} from '@mongodb-js/compass-components';
+
+import type { RootState } from '../../../modules';
+import { runAggregation } from '../../../modules/aggregation';
+import { changeWorkspace } from '../../../modules/workspace';
+import { toggleOptions } from '../../../modules/options';
+
+const containerStyles = css({
+  display: 'flex',
+});
```

Comment:
This needs a `flex: none` or `flex-shrink: 0` disabled (you should generally do this every time you use flex just for positioning and don't want containers to change their size) because right now it starts to collapse when you have a decent amount of pipeline stages:

<img width="266" alt="image" src="https://user-images.githubusercontent.com/5036933/158765629-7d213b13-1aed-4ec3-aca2-ebbdcd687618.png">

Reply:
Switched to grid. And the results are much better. Thanks